File: Python/61-70/e_66.py
# Escribe una función en Python unionLista(L1,L2,L3) que toma como parámetros las listas de enteros L1, L2, y L3, 
# y devuelva una lista en orden asccendente de estas listas sin repetición de números, 
# los números repetidos duplicados serán omitido.

def unionLista(L1, L2, L3):
    # Combinar todas las listas en una sola y eliminar duplicados
    union = list(set(L1 + L2 + L3))
    # Ordenar la lista resultante en orden ascendente
    union.sort()
    return union

# Ejemplo de uso:
L1 = [1, 3, 5, 7, 9]
L2 = [2, 4, 6, 8, 10]
L3 = [1, 2, 3, 4, 5]
resultado = unionLista(L1, L2, L3)
print(resultado)


# Esta función combina todas las listas en una sola lista utilizando el operador +,
# luego utiliza set() para eliminar los elementos duplicados y convierte el resultado nuevamente en una lista.
# Finalmente, la lista resultante se ordena en orden ascendente utilizando sort().

File: JS/61-70/e_66.js
function unionLista(L1, L2, L3) {
    // Combina todas las listas en una sola y elimina duplicados
    var union = [...new Set([...L1, ...L2, ...L3])];
    // Ordena la lista resultante en orden ascendente
    union.sort((a, b) => a - b);
    return union;
}

// Ejemplo de uso:
var L1 = [1, 3, 5, 7, 9];
var L2 = [2, 4, 6, 8, 10];
var L3 = [1, 2, 3, 4, 5];
var resultado = unionLista(L1, L2, L3);
console.log(resultado);



// Esta función combina todas las listas en una sola usando el operador de propagación (...) y el conjunto Set para eliminar los duplicados.
// Luego, ordena la lista resultante en orden ascendente utilizando el método sort().